transport/transport.ts

```
@@ -306,38 +422,43 @@ export abstract class Transport<ConnType extends Connection> {
           msg,
         )}`,
       );
-      return msg.id;
+      return false;
```

what happens to the message in this case?

---

silently discarded, closed transports are not meant to be reopened